hicommonwealth/commonwealth
# User-Onboarding: Show welcome onboard modal for new SSO accounts

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6749

## Description of Changes
- Shows welcome onboard modal for new SSO accounts

## "How We Fixed It"
N/A

## Test Plan

- Enable `FLAG_USER_ONBOARDING_ENABLED` in `.env`.
- Add `type='create-account'` to `<AuthModal/>` instance in `SubLayoutHeader.tsx` or apply this diff
```diff
diff --git a/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx b/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
index 4564d26340..c58647b700 100644
--- a/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
+++ b/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
@@ -59,6 +59,7 @@ export const SublayoutHeader = ({
         open={isFeedbackModalOpen}
       />
       <AuthModal
+        type='create-account'
         onClose={() => setIsAuthModalOpen(false)}
         isOpen={isAuthModalOpen}
       />

```
- Create a new SSO account (google/discord/twitter/apple/github or email) and verify you see the welcome onboard modal showup. The modal will only showup for the first few (5) minutes, since the only requirement of the modal is username which all sso services provide (excluding direct email login, which will be done separately in https://github.com/hicommonwealth/commonwealth/issues/6646)

## Deployment Plan
N/A

## Other Considerations
N/A
